template/src/components/NextImage/NextImage.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+
+type NextImageProps = {
+  src: string;
+  altText: string;
+  className?: string;
+};
+
+export default function NextImage({ src, altText, className }: NextImageProps) {
```

Comment:
Side note for next PR, we need to start move the NFT Image stuff inside the OnchainKit folder.
As ideally we need to start building a NFT Media component we can abstract into the OnchainKit.

Reply:
Good point, that makes sense.